### PR TITLE
Made wording more welcoming

### LINKED ISSUE #522: Simplify language in translation page
In https://devguide.python.org/documenting.html#i-have-a-translation-but-not-on-git-what-should-i-do it says:

"Just ask for help on the doc-sig mailing list and our python-fu, git-fu and bash-fu combined will help you create an appropriate repository. If you use a tool like transifex don’t worry. Keeping them in sync is not that hard."

That answer sounds a little demeaning to me.

- I would suggest not using terms like git-fu
- intsead of _"just ask for help on ..."_, use phrase like _"Ask for help on ..."_ or _"you can ask for help on ...."_
- Instead of _"if you use tool like transifex don't worry ... "_, we can say _"Using a tool like ... can help keeping them in sync"_

### PR DESCRIPTION
**Pull request by jablonskidev**
Closes #522

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).



## CLA Missing

Our records indicate the following people have not signed the CLA:

@jablonskidev

For legal reasons we need all the people listed to sign the CLA before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: documenting.rst [hunk 1]
```diff
@@ -1792,10 +1792,7 @@ Ask on doc-sig, or better, make a PR on the `devguide
 I have a translation, but not on git, what should I do?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Just ask for help on the doc-sig mailing list and our python-fu, git-fu
-and bash-fu combined will help you create an appropriate repository. If
-you use a tool like transifex don’t worry.  Keeping them in sync is not
-that hard.
+You can ask for help on the doc-sig mailing list, and the team will help you create an appropriate repository. You can still use tools like transifex, if you like.
```
Comment (Mariatta) [inline documenting.rst:1795]: I think the new text sounds much better than the existing one. Thank you!

My only suggestion is to keep the text max 80 characters each line.

```suggestion
You can ask for help on the doc-sig mailing list, and the team will help you
create an appropriate repository. You can still use tools like transifex,
if you like.
```

**Review by Mariatta — APPROVED** [review]

**Comment by jablonskidev** [comment]
@Mariatta Excellent point! I'll keep that in mind for next time.

**Comment by Mariatta** [comment]
Thanks for the PR!

**Pull request merged**